src/Workspaces/Core/Portable/Workspace/ProjectSystem/ProjectSystemProject.BatchingDocumentCollection.cs

```diff
@@ -534,24 +534,24 @@ public void ReorderFiles(ImmutableArray<string> filePaths)
         /// While it is OK for this method to *read* local state, it cannot *modify* it as this may
         /// be called multiple times (when the workspace update fails due to interceding updates).
         /// </summary>
-        internal void UpdateSolutionForBatch(
+        internal ImmutableArray<(DocumentId documentId, SourceTextContainer textContainer)> UpdateSolutionForBatch(
```

Comment:
doc the return type of this.  it was not apparent to me that this is the docs you'll open.